src/Core/JVM/Invoker/Invokable.php

```diff
@@ -82,6 +85,8 @@ function () use ($name, $arguments) {
             }
         );
 
+        $currentConstantPool = $method->getConstantPool() ?? $currentConstantPool;
```

Comment:
A method has a unique Constant Pool reference.
Therefore, by adding this code, the method can be refering the Constant Pool in targeted class.